### PR TITLE
feat(ActivityCenter): Support status-go PR with endpoints refactor

### PR DESCRIPTION
**Pull request by MishkaRogachev**
### What does the PR do

Switch on new endpoints : https://github.com/status-im/status-go/pull/3237

### Affected areas

Activity Center

### CODE REVIEW

File: src/app_service/service/activity_center/service.nim [hunk 1]
```diff
@@ -51,6 +51,15 @@ const SIGNAL_MARK_NOTIFICATIONS_AS_DISMISSED* = "markNotificationsAsDismissed"
 
 const DEFAULT_LIMIT = 20
 
+# NOTE: temporary disable Transactions and System and we don't count All group
```
Comment (caybro) [inline src/app_service/service/activity_center/service.nim:54]: Is there an issue tracking this? If so, pls mention it here

File: src/backend/backend.nim [hunk 2]
```diff
@@ -46,6 +46,16 @@ type
     chainColor* {.serializedFieldName("chainColor").}: string
     shortName* {.serializedFieldName("shortName").}: string
 
+  ActivityCenterNotificationsRequest* = ref object of RootObj
+    cursor* {.serializedFieldName("cursor").}: string
+    limit* {.serializedFieldName("limit").}: int
+    activityTypes* {.serializedFieldName("activityTypes").}: seq[int]
+    readType* {.serializedFieldName("limit").}: int
```
Comment (caybro) [inline src/backend/backend.nim:53]: "limit"?

File: src/backend/backend.nim [hunk 3]
```diff
@@ -46,6 +46,16 @@ type
...
+
+  ActivityCenterCountRequest* = ref object of RootObj
+    activityTypes* {.serializedFieldName("activityTypes").}: seq[int]
+    readType* {.serializedFieldName("limit").}: int
```
Comment (caybro) [inline src/backend/backend.nim:57]: "limit"?

File: src/backend/backend.nim (hunk 2 above)
Comment (MishkaRogachev) [reply src/backend/backend.nim]: fixed

File: src/backend/backend.nim (hunk 3 above)
Comment (MishkaRogachev) [reply src/backend/backend.nim]: fixed

File: src/app/modules/main/activity_center/module.nim [hunk 4]
```diff
@@ -303,23 +303,5 @@ method setActivityCenterReadType*(self: Module, readType: int) =
 method getActivityCenterReadType*(self: Module): int =
   return self.controller.getActivityCenterReadType().int
 
-method groupCountersChanged*(self: Module) =
-  self.view.groupCountersChanged()
-
-method getAdminCount*(self: Module): int =
```
Comment (endulab) [inline src/app/modules/main/activity_center/module.nim:309]: You removed a bunch of methods here but they are still in io_interface

**Review by endulab — CHANGES_REQUESTED** [review]

File: src/app_service/service/activity_center/service.nim (hunk 1 above)
Comment (caybro) [reply src/app_service/service/activity_center/service.nim:54]: OK

File: src/app/modules/main/activity_center/module.nim (hunk 4 above)
Comment (MishkaRogachev) [reply src/app/modules/main/activity_center/module.nim:309]: Oh, i expected it won't compile with unimplemented methods. Fixed :)

**Review by caybro — APPROVED** [review]
LGTM

**Review by endulab — APPROVED** [review]

**Pull request merged**